templates/base.html.twig

```diff
@@ -110,8 +70,6 @@
 </div>
 <div class="footer">
     <div class="footer-inner">
-        <span><a href="https://wiki.surfnet.nl/display/conextsupport/Terms+of+Service+%28NL%29"
-                 target="_blank">{{ 'page.footer.terms'|trans }}</a></span>
```

Comment:
No need to remove this section in my opinion

Reply:
Do you perhaps know why it was removed from MFA?

Reply:
Thanks for the clarification, I will restore it